### ISSUE TITLE
Document Viewer  | Accessibility: New hover color

### ISSUE THREAD

**Issue by gnakm** [department-of-veterans-affairs/caseflow#2160]
**Description**
Our hover color on toolbar failed 508 :( This is a new color, now being added to the style guide, that passes WCAG AA.

## AC
- [x] Verify that the new hover color is #02bfe7 `primary-alt` Not in style guide yet but can be found in this [issue](https://github.com/department-of-veterans-affairs/caseflow/issues/2159)
- [x] Verify that the following now has new hover color  
    - Back to all documents 
    - Doc type name 
    - Zoom out, fit to screen, in 
    - Previous, next
    - add comment document highlight

Not in style guide yet but link to it:

**Comment by Sjones352** [comment]
<img width="1271" alt="screen shot 2017-06-19 at 8 37 42 pm" src="https://user-images.githubusercontent.com/23080951/27311507-550a6474-552f-11e7-89bf-b6f96a2b9cc8.png">

**Comment by astewarttistatech** [comment]
**PASSED**

Verified in: UAT
Browsers: FF and Chrome

Steps to Validate: 
1. Access Reader
2. Add a comment to a document
3. While the hover is active, use the digital color meter (Mac) to locate the RGB and convert it to HEX
4. Verify the hex colors match. 

Screenshots: 
<img width="1304" alt="screen shot 2017-06-20 at 11 35 58 am" src="https://user-images.githubusercontent.com/23038505/27342152-48aeb710-55ad-11e7-94e8-252eefb54f7a.png">
![colorhover](https://user-images.githubusercontent.com/23038505/27342162-4e598da2-55ad-11e7-9cbe-62b646c84ff1.gif)
